public/app/features/explore/Logs/Logs.tsx

```diff
@@ -552,6 +574,23 @@ class UnthemedLogs extends PureComponent<Props, State> {
           )}
         </PanelChrome>
         <PanelChrome
+          titleItems={[
+            config.featureToggles.logsExploreTableVisualisation ? (
+              this.state.visualisationType === 'logs' ? null : (
+                <PanelChrome.TitleItem title="Experimental" key="A">
+                  <Tag
```

Comment:
There is `<FeatureBadge featureState={FeatureState.beta} />`, so we don't need to make our own.

Reply:
Good catch, fixed!